engine/src/main/java/org/terasology/engine/utilities/ReflectionUtil.java

```diff
@@ -265,6 +265,7 @@ public static Method findMethod(Class<?> targetType, String methodName, Class<?>
         return (TypeInfo<E>) TypeInfo.of(elementType);
     }
 
+    @SuppressWarnings("checkstyle:ParameterAssignment")
```

Comment:
Maybe we should add a comment here why this is suppressed.

In this case, we're not sure whether the reassignment is necessary, and/or whether it is a desired property of this method.